python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+    crosstalkSources = cT.PrerequisiteInput(
+        name="isrOverscanCorrected",
+        doc="Overscan corrected input images.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+        deferLoad=True,
+        multiple=True,
+        lookupFunction=crosstalkSourceLookup,
+        minimum=0,  # not needed for all instruments, no config to control this
+    )
+    defects = cT.PrerequisiteInput(
+        name='defects',
+        doc="Input defect tables.",
+        storageClass="Defects",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    bfKernel = cT.PrerequisiteInput(
+        name='brighterFatterKernel',
+        doc="Newer complete kernel + gain solutions.",
+        storageClass="BrighterFatterKernel",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+        minimum=0,  # can use either bfKernel or newBFKernel
+    )
+    dark = cT.PrerequisiteInput(
+        name='dark',
+        doc="Input dark calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    outputExposure = cT.Output(
+        name='postISRCCD',
+        doc="Output ISR processed exposure.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    preInterpExposure = cT.Output(
+        name='preInterpISRCCD',
+        doc="Output ISR processed exposure, with pixels left uninterpolated.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    outputStatistics = cT.Output(
+        name="isrStatistics",
+        doc="Output of additional statistics table.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+
+    def __init__(self, *, config=None):
+        super().__init__(config=config)
+
+        if config.doDiffNonLinearCorrection is not True:
+            self.prerequisiteInputs.remove("dnlLUT")
+        if config.doBias is not True:
+            self.prerequisiteInputs.remove("bias")
+        if config.doDeferredCharge is not True:
+            self.prerequisiteInputs.remove("deferredChargeCalib")
+        if config.doLinearize is not True:
+            self.prerequisiteInputs.remove("linearizer")
+        if config.usePtcGains is not True and config.usePtcReadNoise is not True:
+            self.prerequisiteInputs.remove("ptc")
+        if config.doCrosstalk is not True:
+            self.prerequisiteInputs.remove("crosstalkSources")
+            self.prerequisiteInputs.remove("crosstalk")
+        if config.doDefect is not True:
+            self.prerequisiteInputs.remove("defects")
+        if config.doBrighterFatter is not True:
+            self.prerequisiteInputs.remove("bfKernel")
+        if config.doDark is not True:
+            self.prerequisiteInputs.remove("dark")
+
+        if config.doWrite is not True:
```

Comment:
Is `doWrite` actually something we want to support, or is this a hangover from the way tasks were written in the gen2 era?  (I'm pretty sure this is not a config we want).